src/airflow/dags/common_airflow.py

```diff
@@ -86,7 +86,7 @@ def create_cluster(
         zone=GCP_ZONE,
         master_machine_type=master_machine_type,
         worker_machine_type=worker_machine_type,
-        master_disk_size=500,
+        master_disk_size=2000,
```

Comment:
Do we need this size by default or is it only useful for Finngen?

Reply:
parametrised in a new commit